test/tests/parallel/set3/scripts/postconf-helper.bats

```diff
@@ -0,0 +1,61 @@
+load "${REPOSITORY_ROOT}/test/helper/common"
+load "${REPOSITORY_ROOT}/test/helper/setup"
+
+BATS_TEST_NAME_PREFIX='[Scripts] (helper functions) (postfix.sh) (_add_to_or_update_postfix_main) '
+CONTAINER_NAME='dms-test-main.cf-changes'
```

Comment:
Convention is `dms-test` + container name + optional variant, where each value is kebab-case with `_` separator. Probably better to avoid `.` in the name too 🤷‍♂️ 

Let's just align with the test filename in this case?

```suggestion
CONTAINER_NAME='dms-test_postconf-helper'
```